src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -100,4 +100,13 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+  
+  /** Returns the amount of conversations currently in the database. */
+  public int getConversationAmount(){
+    if(this == null){
+      return 0;
+    }
```

Comment:
Fix the spacing in this method please. Space always after () and in if statements

```java
public int getConversationAmount() {
  if (this == null) {
    return 0;
  }
}